uber/utils.py

```diff
@@ -262,17 +262,30 @@ def charge_cc(self, token):
             )
         except stripe.CardError as e:
             return 'Your card was declined with the following error from our processor: ' + str(e)
-        except stripe.error.InvalidRequestError as e:
-            log.error('Invalid source object provided', exc_info=True)
-            send_email(c.ADMIN_EMAIL, [c.ADMIN_EMAIL, 'dom@magfest.org'], 'MAGFest Stripe invalid request error',
-                       'Got an error while calling charge_cc(self, token={!r}):\n{}'
-                       .format(token, traceback.format_exc()))
-            return 'An invalid source object was provided: ' + str(e)
         except stripe.StripeError as e:
-            log.error('unexpected stripe error', exc_info=True)
+            error_txt = 'Got an error while calling charge_cc(self, token={!r})'.format(token)
+            report_critical_exception(msg=error_txt, subject='ERROR: MAGFest Stripe invalid request error')
```

Comment:
@migetman9 FYI, I kinda removed the section specific to InvalidRequestError because now we're catching all exceptions via StripeError (of which InvalidRequestError is one of them).

This also display a bit of a nicer error message for the end user